controller/index.js

```diff
@@ -1,16 +1,30 @@
 
 var pkg = require('../package');
+var markdown = require('markdown').markdown;
+var fs = require('fs');
 
 function setup(){
 
   function controller( req, res, next ){
 
-    // stats
-    res.json({
-      name: pkg.name,
-      version: {
-        number: pkg.version
+    fs.readFile('./DOCS.md', 'utf8', function (err, content) {
```

Comment:
This is going to hit disk every time, it might be better to load the file in RAM once when the server starts up since it's a small file that might be served often.